src/persist-client/src/internal/apply.rs

```diff
@@ -207,6 +207,28 @@ where
             })
     }
 
+    /// WIP:
+    pub fn removable_rollups(&self) -> Vec<(SeqNo, PartialRollupKey)> {
+        let mut rollups: Vec<(SeqNo, PartialRollupKey)> =
+            self.state
+                .read_lock(&self.metrics.locks.applier_read_noncacheable, |state| {
+                    state
+                        .collections
+                        .rollups
+                        .range(..=state.seqno_since())
```

Comment:
as we discussed in our 1:1, I think we need to make the end of the range exclusive

Reply:
hm I actually don't remember this discussion 😅 do you still think this is true given the `pop()` after / https://github.com/MaterializeInc/materialize/pull/19384/files#r1201156588?

Reply:
whoops, wrote this comment before I saw the pop, and meant to replace it with the one i left below https://github.com/MaterializeInc/materialize/pull/19384#discussion_r1201156588